pkg/validations/runner.go

```diff
@@ -6,12 +6,15 @@ var errRunnerValidation = errors.New("validations failed")
 
 type Validation func() *ValidationResult
 
+type ValidationFunc func() error
```

Comment:
what is this for?

Reply:
This was left by mistake from some previous changes I made. Removing!